torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1260,6 +1260,28 @@ def reference_sigmoid(x):
             return (1 / (1 + np.exp(-x)))
         return scipy.special.expit(x)
 
+    def reference_lgamma(x):
+        # scipy.special.gammaln returns `-inf` when input is `-inf`.
+        # While Pytorch, C and C++, all return `inf` when input is `-inf`.
```

Comment:
This is a good comment. Would you extend it to include Python's math.lgamma, too?